desktop.css

```diff
@@ -0,0 +1,925 @@
+@media screen and (min-width: 720px) {
```

Comment:
- Hey, please, let's increase the breakpoint to be greater than 720 to start at least from 768p above to meet the project requirement.